opencensus/exporters/stats/prometheus/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+opencensus_lib(exporters_stats_prometheus_exporter
```

Comment:
IMO let's call this `exporters_stats_prometheus` because it's less stutter and it is consistent with the existing `exporters_stats_stdout`